### ISSUE TITLE
Feature Request: Canvas Interaction

### ISSUE THREAD

**Issue by hrstwn** [brain-child/moobo#18]
These are probably hard to implement but can improve the usability of the apps a lot.
This kind of apps (in my use case) needs to have some canvas interaction feature to maximize the use of the drawing area.
Some control that users may expect are scrolling/panning and zooming. Not all applications alike has zooming feature but being able to scroll or pan through canvas is somewhat necessary. That way users aren't bounded by the window/screen size to use the app.

Other elementary app that can be used as an example is Minder. I think it handles canvas interaction really well. It supports panning/scrolling, zooming, even focusing to certain node/object.

Here's a screencast of the app:
https://imgur.com/izjmODL

**Comment by pierrefabarius** [comment]
Yes, this was discussed before and would be the ultimate goal for the app. But as you say, this can probably be very challenging and takes some time. I would love to have at least the scrolling option in version 1.0.
